scripts/monitoring/cron-send-security-updates-count.py

```diff
@@ -81,8 +83,15 @@ def get_security_updates(self):
         if proc.returncode != 0:
             self.sec_updates = []
             raise subprocess.CalledProcessError(proc.returncode, subprocess.list2cmdline(cmd), output=stdout+stderr)
-        lineslist = [line for line in stdout.split('\n') if line and 'Errno' not in line]
-        self.sec_updates = [UpdateItem(*re.split(r'\s+', line)) for line in lineslist]
+        for line in stdout.split('\n'):
+            if line and 'Errno' not in line:
+                if len(re.split(r'\s+', line)) == 3:
+                    self.sec_updates.append(UpdateItem(*re.split(r'\s+', line)))
+                else:
+                    logging.error('Security update %s looks like error output', line)
```

Comment:
This'll make for unreadable log output with the unrecognized line stuck in the middle:

logging.error('Security output looks like an error:)
logging.error('- %s', line)